image/docker_schema2.go

```diff
@@ -176,18 +181,26 @@ func (m *manifestSchema2) UpdatedImage(options types.ManifestUpdateOptions) (typ
 	return memoryImageFromManifest(&copy), nil
 }
 
-func (m *manifestSchema2) convertToManifestOCI1() (types.Image, error) {
+func (m *manifestSchema2) OCIConfigFromDocker() (*imgspecv1.Image, error) {
```

Comment:
This is a private helper, so please name it starting with a lowercase letter.

Or perhaps just move the code into the public `OCIConfig` and call `m.OCIConfig` from `convertToManifestOCI1`; one less function.